modules/dummy-module-xyz.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-the-profile-creator-tool"]
```

Comment:
```suggestion
[id="cnf-about-the-profile-creator-tool_{context}"]
```

Reply:
{context} is missing
Context used for identifying headers in modules that is the same as the anchor ID. Example: cli-developer-commands. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata